### PR TITLE
NVyrsta

### PR DESCRIPTION
**Pull request by NVyrsta**

### CODE REVIEW

File: src/createServer.js [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
+    res.setHeader('Content-Type', 'application/json');
+
+    const normalizeURL = new URL(req.url, `http://${req.headers.host}`);
+
+    const targetCase = normalizeURL.searchParams.get('toCase');
+    const originalText = normalizeURL.pathname.slice(1);
+
+    const errors = handleErrors(originalText, targetCase);
+
+    if (errors.length > 0) {
```
Comment (Dashaverz) [inline src/createServer.js:16]: `if (errors.length) {`

File: src/createServer.js [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+
+      res.end(JSON.stringify({
+        errors,
+      }));
+
+      return;
+    }
+
+    res.statusCode = 200;
+    res.statusMessage = 'OK';
```
Comment (Dashaverz) [inline src/createServer.js:28]: I think res.statusCode is set to 200 and the res.statusMessage is set to OK by default

**Review by Dashaverz — APPROVED** [review]
GJ! left some comments

File: src/createServer.js [hunk 3]
```diff
@@ -1,3 +1,40 @@
...
+      res.statusCode = 404;
+
+      res.end(JSON.stringify({
+        errors,
+      }));
+
+      return;
+    }
+
+    res.statusCode = 200;
```
Comment (alexandra-protyanova) [inline src/createServer.js:26]: status code = 200, is default as well)

**Review by alexandra-protyanova — APPROVED** [review]
Good job 👍

**Pull request closed without merging**